.gitignore

```diff
@@ -5,4 +5,7 @@
 # Rendered html files
 /html
 # Doc trees
-/doctrees
\ No newline at end of file
+/doctrees
+
+# All build files
+/build
```

Comment:
this is not applicable anymore since we have the build inside the generated_docs